src/DoctrineORMModule/Options/DBALConnection.php

```diff
@@ -204,4 +209,20 @@ public function getWrapperClass()
     {
         return $this->wrapperClass;
     }
+
+    /**
+     * @return bool
+     */
+    public function useSavepoints()
+    {
+        return $this->useSavepoints;
+    }
+
+    /**
+     * @param bool $useSavepoints
+     */
+    public function setUseSavepoints($useSavepoints)
+    {
+        $this->useSavepoints = $useSavepoints;
```

Comment:
Add return $this for a fluent interface.

Reply:
@TomHAnderson This class has no fluent interface at all